controllers/renovate_util.go

```diff
@@ -32,6 +32,7 @@ const (
 
 type installationStruct struct {
 	id           int
+	component    string
 	token        string
 	repositories []renovateRepository
 }
```

Comment:
Hmm, so what is an "installation"? Why does it have just one `component` but a list of `repositories`?

Reply:
well it isn't exactly like WHOLE installation struct

but installation for component (1 repository) struct


so 1 struct has info about installation and specific repository for component 

and list of repositories has in fact only 1 repository ......